docs/examples/spinner/main.js

```diff
@@ -0,0 +1,21 @@
+// @flow strict
+import { Fragment, type Node, useState } from 'react';
+import { Box, Button, Spinner } from 'gestalt';
+
+export default function Example(): Node {
+  const [show, setShow] = useState(true);
+
+  return (
+    <Fragment>
+      <Box padding={2}>
+        <Button
```

Comment:
Not sure if we need the Button hide/show functionality in the main example

Reply:
As with the other: agreed that it's not a pattern we use much, but this is how the docs have been